### PR TITLE
Make it easy for people to play the games

### PR DESCRIPTION
**Pull request by wbourne0**
I made a script that uses your script and lets the user play the games on repl.it, this will direct the user to it.

### CODE REVIEW

**Comment by Seniru** [comment]
Hmm... Nice suggestion!

**Comment by Seniru** [comment]
@AllAwesome497 thanks for contributing! This idea is awesome as it let people to play the games without worrying about cloning this repo to their machines.

I tested this game with repl.it and I found some problems with this. It doesn't load the game correctly and throws an error.

Here is an example

```sh
what game
Hangman
MazeEscape
TextAdventures
TicTacToe
Hangman
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module '/home/runner/Hangman/index.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
exit status 1
```

Hope you could fix this problem. Also make the prompt more friendly (I think you know what I mean :P). Otherwise this is good! 👏

**Review by Seniru — CHANGES_REQUESTED** [review]
Changes requested. More info can be found in the conversation

**Comment by wbourne0** [comment]
Let me try and fix it

**Comment by wbourne0** [comment]
Ok i fixed it, the only thing I have added to it is a simple bash script to let the user choose which game they want to play (I also renamed a few files to simplify it)

**Comment by wbourne0** [comment]
Here is the bash script in case you want to see it:
```bash
npm i prompts; npm i ansi-escapes; echo Which game do you want to play?; echo Hangman; echo MazeEscape; echo TextAdventures; echo TicTacToe; read game; node $game/index.js```

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # cli-games
-Games in the terminal
+> Games in the terminal
+
+[![Run on Repl.it](https://repl.it/badge/github/salifm/cli-games)](https://repl.it/@AllAwesome497/cli-games)
```
Comment (Seniru) [inline README.md:4]: I suppose this should link to the main repo (salif's one)

**Review by Seniru — CHANGES_REQUESTED** [review]
I think the badge should link to the repl of this repository. Other than that the work is perfect!
I think you can fix this small problem and the PR would be merged soon!

**Comment by wbourne0** [comment]
Wdym? [Run on Repl.it](https://repl.it/@AllAwesome497/cli-games) goes straight to a fork of this repository, and it should pull commits from it also.

**Comment by Seniru** [comment]
The badge links to your repository instead of salif's one. That should be corrected ☺

**Comment by wbourne0** [comment]
As far as I know salif does not have a repl for it. If they would like to i'd be happy to help them set it up though.

**Comment by wbourne0** [comment]
Basically that repl just pulls from this repo.

**Comment by wbourne0** [comment]
Oh and in my repl it still credits salifm, just so u know.

**Review by salif — APPROVED** [review]

**Comment by Seniru** [comment]
I just want to make sure the repl get updated when there are changes to salif's repo. If yes, then this PR will get merged.

**Comment by wbourne0** [comment]
I can say for certain that it works every time if u make a selection menu to run the games. I'll make an addition to my run script later to make sure it automatically pulls and then runs it.

**Comment by Seniru** [comment]
OK! I'll merge this. Inform me after you did the script

**Review by Seniru — APPROVED** [review]
Approved!

**Comment by Seniru** [comment]
Enjoy 🎉🎉🎉

**Pull request merged**